.github/workflows/ci.yml

```diff
@@ -10,8 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Setup Node
-        uses: actions/setup-node@v3
+      - uses: pnpm/action-setup@v2.0.1
```

Comment:
Looks like it's failing because it can't find a version - does it need something like this? 
`uses: pnpm/action-setup@v2.2.2
        with:
          version: 6.0.2`

Reply:
Also should we use the latest version, i.e. v2.2.2?

Reply:
ah yeah, i think later version will use the `packageManager` key in the `package.json`. lazy copying :)